packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -308,7 +310,12 @@ const getAppealType = (appealCaseData) => {
 	if (isV2Submission(appealCaseData)) {
 		return getAppealTypeNameByTypeCode(appealCaseData?.AppellantSubmission?.appealTypeCode);
 	}
-	return `${appealCaseData.appealTypeName} appeal`;
+
+	const caseType =
+		appealCaseData.appealTypeCode in CASE_TYPES
+			? CASE_TYPES[appealCaseData.appealTypeCode].type
+			: '';
```

Comment:
Maybe this needs a little shared util as well? `caseTypeNameWithDefault` or something